plugins/serializers/json/json_test.go

```
@@ -23,7 +29,7 @@ func TestSerializeMetricFloat(t *testing.T) {
 	}
 	m := metric.New("cpu", tags, fields, now)
 
-	s, _ := NewSerializer(0, "")
+	s, _ := NewSerializer(0, "", "")
```

The returned error is ignored throughout the test, think we should check it?